tools/src/component-generator.test.ts

```diff
@@ -3,7 +3,8 @@ import generate from "./component-generator";
 it("generates", () => {
     //#region EXPECTED
     const EXPECTED = `
-import dxCLASS_NAME, { IOptions as ICLASS_NAMEOptions } from "devextreme/DX/WIDGET/PATH";
+import dxCLASS_NAME,
+       { IOptions as ICLASS_NAMEOptions } from "devextreme/DX/WIDGET/PATH";
```

Comment:
What about moving the `from` statement to a new line?

Reply:
Yes, I think about it, but "from" position controlled by [the import-spacing rule](https://palantir.github.io/tslint/rules/import-spacing/). We should disable this rule to move "from". I don't think that's right.

Reply:
I see... what about this?
```ts
import dxLinearGauge, {
    IOptions as ILinearGaugeOptions
} from "devextreme/viz/linear_gauge";
```